service-runner/src/main/resources/config/assets.yaml

```diff
@@ -190,7 +190,7 @@ assets:
   # Fiat
   - schema: iso4217
     code: USD
-    significant_decimals: 7
+    significant_decimals: 4
```

Comment:
And fiat should be 2

Reply:
Should we fix reference server impl?

Reply:
I think `4` is good. We need to make AP and reference server match. It is also good to try different decimals.